incubator/hnc/Makefile

```diff
@@ -182,21 +182,17 @@ kind-deploy:
 	CONFIG=kind $(MAKE) deploy
 
 ###################### E2E TEST #########################
-# Sleep time is added for the webhook to get ready before test. The time 
-# comes from watching the log as well as multiple experiments. 
+# This test will run on the current cluster the user deployed (either kind or 
+# a kubernetes cluster). Please make sure to have the test cluster ready 
```

Comment:
Don't need to have the "please make sure" in the comment since it's displayed to the user, which is a much more visible spot.

Also, don't put a blank line between the comment and the rule.

Reply:
Done